### PR TITLE
Search working

### PR DESCRIPTION
**Pull request by stormwild**

### CODE REVIEW

File: angular.json [hunk 1]
```diff
@@ -132,5 +132,10 @@
       }
     }
   },
-  "defaultProject": "school-rankings-app"
+  "defaultProject": "school-rankings-app",
+  "schematics": {
+    "@schematics/angular:component": {
+      "styleext": "scss"
```
Comment (stormwild) [inline angular.json:138]: This allows styling using sass.

File: src/app/app-routing.module.ts [hunk 2]
```diff
@@ -31,7 +31,7 @@ const routes: Routes = [
     path: '**',
     component: PageNotFoundComponent
   }
-  
+
```
Comment (stormwild) [inline src/app/app-routing.module.ts:34]: These are just formatting changes due to applying jshint.

File: angular.json (hunk 1 above)
Comment (bhansaliraunak) [reply angular.json:138]: that'd be grt

**Review by bhansaliraunak — APPROVED** [review]
@stormwild Everything looks fine!

**Pull request merged**